plugins/filter/counter.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021, Remy Keil <remy.keil@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleFilterError
+from ansible.module_utils.common._collections_compat import Sequence
+from collections import Counter
+
+
+def counter(sequence):
+    ''' Count elements in a sequence. Returns dict with count result. '''
+    if not isinstance(sequence, Sequence):
+        raise AnsibleFilterError('First argument for counter must be a sequence (str or list). %s is %s' %
+                                 (sequence, type(sequence)))
+
+    for element in sequence:
+        try:
+            hash(element)
```

Comment:
Since `Counter()` already does this: how about adding a `try`/`except TypeError` around line 26 instead?

Reply:
Indeed, that's much nicer. Fixed with https://github.com/ansible-collections/community.general/pull/3921/commits/cc77c72b4398245baac8e8ded819101a8dbb886d